spec/takeaway_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'takeaway'
+
+describe Takeaway do
+    
+    # As a customer
+    # So that I can check if I want to order something
+    # I would like to see a list of dishes with prices
+    let(:dishes) {[
+        {dish: 'Lasagne', price: 4.99},
+        {dish: 'Chicken', price: 5.99},
+        {dish: 'Steak', price: 8.99},
+        {dish: 'Burger', price: 7.99},
+        {dish: 'Borritto', price: 7.99},
+        {dish: 'Omelette', price: 4.99}
+    ]}
```

Comment:
👍🏻  Good use of mock for menu class  - 

When working with many classes and files, you want them to run tests independently - that way [debugging](https://github.com/makersacademy/course/blob/master/goals/course_goals.md#i-can-make-software) will be a lot easier. So, by mocking the menu, you've given your takeaway_spec all the information it would get from the menu class - without it having to ask the menu class. Hooray!